openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx

```diff
@@ -34,7 +34,7 @@ const QueryCard: FC<QueryCardProp> = ({ className, query }) => {
         className
       )}
       onClick={() => setExpanded((pre) => !pre)}>
-      {!isUndefined(query.user) && !isUndefined(query.duration) ? (
+      {/* {!isUndefined(query.user) && !isUndefined(query.duration) ? (
```

Comment:
opened a seperate issue to handle users in ui https://github.com/open-metadata/OpenMetadata/issues/8475